semgrep-core/tests/generate_test_matrix.py

```diff
@@ -65,7 +65,7 @@
     ],
 }
 
-EXCLUDE = ["TODO", "GENERIC", "fuzzy", "lint", 'EQUIV', 'e2e', 'SYNTHESIZING', 'NAMING', 'PERF', 'TAINTING']
+EXCLUDE = ["TODO", "GENERIC", "fuzzy", "lint", 'EQUIV', 'e2e', 'SYNTHESIZING', 'NAMING', 'PERF', 'TAINTING', 'EVAL']
```

Comment:
Maybe i should put all those dirs under a NOT_A_LANG durectory

Reply:
yeah that's probably a good idea, or put the languages under lang/ or something